software/portkey.yml

```diff
@@ -0,0 +1,12 @@
+name: "portkey"
+website_url: "https://portkey.page"
+source_code_url: "https://github.com/kodehat/portkey"
+description: "A simple web portal that can act as startup page and shows a collection of links/urls. Also supports adding custom pages. Everything with one config file!"
```

Comment:
```suggestion
description: "A simple web portal that can act as startup page and shows a collection of links/urls. Also supports adding custom pages. Everything with one config file."
```

Reply:
Hi,
sorry that I did not see that the linting workflow had failed. I have updated my commit with your suggestion and force pushed the branch.